lib/fetching/bot.js

```diff
@@ -34,14 +34,15 @@ util.inherits(Bot, EventEmitter);
 Bot.prototype.start = function() {
   if (this.enabled) return;
   this.enabled = true;
+  this.source.enabled = true;
```

Comment:
This line worries me. Seems like we're breaking an abstraction if a bot can modify its source (especially without checking in with the db!)